src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,22 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard
+        imgUrl={movie.imgUrl}
+        title={movie.title}
+        description={movie.description}
+        imdbUrl={movie.imdbUrl}
+        key={movie.imdbId}
```

Comment:
In this case, it will be better and easier to pass all the values as a single prop - `movie'.

Reply:
after that, all tests will be passed)

<img width="484" alt="image" src="https://user-images.githubusercontent.com/86183007/225049855-32330d42-8f16-4aa6-a72f-3ddad27c1245.png">